atmo/appsource/appsource.go

```diff
@@ -13,37 +13,48 @@ var (
 	ErrAuthenticationFailed = errors.New("failed to authenticate")
 )
 
-// Meta describes the metadata for an App
+// Meta describes the metadata for an App.
 type Meta struct {
 	Identifier string `json:"identifier"`
 	AppVersion string `json:"appVersion"`
+	Domain     string `json:"domain"`
 }
 
 type AppSource interface {
-	// Start indicates to the AppSource that it should prepare for app startup
-	Start(options.Options) error
-	// Runnables returns all of the available Runnables
-	Runnables() []directive.Runnable
+	// Start indicates to the AppSource that it should prepare for app startup.
+	Start(opts options.Options) error
+
+	// Runnables returns all of the available Runnables.
+	Runnables(ident, version string) []directive.Runnable
+
 	// FindRunnable directs the AppSource to attempt to find
```

Comment:
This comment actually isn't true, we should just say "FindRunnable attempts to find the given Runnable, and returns ErrRunnableNotFound if it cannot" or something.

Reply:
Reworded in https://github.com/suborbital/atmo/pull/121/commits/799eddbfd83d427d96d9f3336ddc6a91fb73208f